test/test_build.py

```diff
@@ -79,18 +79,22 @@ def image_type_fixture(shared_tmpdir, build_container, request, force_aws_upload
     """
     container_ref, images, target_arch, local = parse_request_params(request)
 
-    if local:
-        cont_tag = "localhost/cont-base-" + "".join(random.choices(string.digits, k=12))
+    if not local:
```

Comment:
I wonder if we could avoid one `if` here by simply (re)using the `container_ref` var (which is probably a slightly better name anyway), something like:
```python
diff --git a/test/test_build.py b/test/test_build.py
index 9413949..4d0d9db 100644
--- a/test/test_build.py
+++ b/test/test_build.py
@@ -79,21 +79,18 @@ def image_type_fixture(shared_tmpdir, build_container, request, force_aws_upload
     """
     container_ref, images, target_arch, local = parse_request_params(request)
 
-    if not local:
-        cont_tag = container_ref
-    else:
-        cont_tag = "localhost/cont-base-" + \
+    if local:
+        container_ref = "localhost/cont-base-" + \
             "".join(random.choices(string.digits, k=12))
-
         # we are not cross-building local images (for now)
-        request.param = ",".join([cont_tag, images, "", "true"])
+        request.param = ",".join([cont_ref, images, "", "true"])
 
     # copy the container into containers-storage
     # for all builds
     subprocess.check_call([
         "skopeo", "copy",
         f"docker://{container_ref}",
-        f"containers-storage:[overlay@/var/lib/containers/storage+/run/containers/storage]{cont_tag}"
+        f"containers-storage:[overlay@/var/lib/containers/storage+/run/containers/storage]{container_ref}"
     ])
 
     with build_images(shared_tmpdir, build_container, request, force_aws_upload) as build_results:
```

Reply:
Oh yeah good suggestion! Thanks